### PR TITLE
Use Scalar DB 3.6.0 in Multi-storage Transaction Sample

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: multi-storage-transaction-sample/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.scalar-labs:scalardb:3.5.2'
+    implementation 'com.scalar-labs:scalardb:3.6.0'
```
Comment (brfrn169) [inline multi-storage-transaction-sample/build.gradle:14]: Use Scalar DB 3.6.0.

File: multi-storage-transaction-sample/src/main/java/sample/Sample.java [hunk 2]
```diff
@@ -23,8 +21,7 @@ public class Sample implements AutoCloseable {
 
   public Sample() throws IOException {
     // Create a transaction manager object
-    TransactionFactory factory =
-        new TransactionFactory(new DatabaseConfig(new File("database.properties")));
+    TransactionFactory factory = TransactionFactory.create("database.properties");
```
Comment (brfrn169) [inline multi-storage-transaction-sample/src/main/java/sample/Sample.java:24]: Also, use the new Scalar DB API.

File: multi-storage-transaction-sample/README.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ The `customers` table is created in the `customer` namespace. And the `orders`,
 The `customer.customers` table manages customers' information.
 The `credit_limit` is the maximum amount of money a lender will allow each customer to spend using a credit card, and the `credit_total` is the amount of money that each customer has already spent by using the credit card.
 
-The `orders.orders` table manages orders' information, and the `orders.statements` table manages the statements' information of the orders. Finally, the `orders.items` table manages items' information to be ordered.
+The `order.orders` table manages orders' information, and the `order.statements` table manages the statements' information of the orders. Finally, the `order.items` table manages items' information to be ordered.
```
Comment (brfrn169) [inline multi-storage-transaction-sample/README.md:90]: Also, refactored README a bit.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**